libyul/CompilabilityChecker.cpp

```diff
@@ -43,27 +43,31 @@ map<YulString, int> CompilabilityChecker::run(
 
 	solAssert(_dialect.flavour == AsmFlavour::Strict, "");
 
-	solAssert(dynamic_cast<EVMDialect const*>(&_dialect), "");
-	NoOutputEVMDialect noOutputDialect(dynamic_cast<EVMDialect const&>(_dialect));
-	BuiltinContext builtinContext;
+	if (EVMDialect const* evmDialect = dynamic_cast<EVMDialect const*>(&_dialect))
+	{
+		NoOutputEVMDialect noOutputDialect(*evmDialect);
+		BuiltinContext builtinContext;
 
-	yul::AsmAnalysisInfo analysisInfo =
-		yul::AsmAnalyzer::analyzeStrictAssertCorrect(noOutputDialect, _ast);
+		yul::AsmAnalysisInfo analysisInfo =
```

Comment:
whitespace here and below.

Reply:
That's because the assignment line would be too long